package.json

```diff
@@ -9,46 +9,46 @@
     "ets": "bin/ets.js"
   },
   "dependencies": {
-    "autod": "^3.1.0",
-    "chalk": "^2.4.2",
+    "autod": "^3.1.1",
+    "chalk": "^4.1.1",
     "co-mocha": "^1.2.2",
     "common-bin": "^2.9.0",
-    "debug": "^4.1.1",
+    "debug": "^4.3.1",
     "detect-port": "^1.3.0",
-    "egg-ts-helper": "^1.25.2",
+    "egg-ts-helper": "^1.25.9",
     "egg-utils": "^2.4.1",
     "espower-source": "^2.3.0",
-    "globby": "^9.2.0",
+    "globby": "^11.0.3",
```

Comment:
globby 11.0.3  >= Node 10,   change to 10.0.2

Reply:
ok